carmm/run/aims_calculator.py

```diff
@@ -276,4 +261,4 @@ def get_k_grid(model, sampling_density, verbose=False, simple_reciprocal_space_p
                   "This would generate a slightly denser k-grid than using simple reciprocal space parameters in "
                   "cases where a non-orthogonal cell is used as input.")
 
-    return k_grid
+    return tuple(k_grid)
```

Comment:
I guess this backwards conversion is to ensure validity with existing infrastructure, i.e. returns the same format as previously but needs to complete conversion as now it has been changed to a numpy array?

Reply:
Yes. The original return value is a tuple, so I convert the array to tuple here.